satpy/tests/reader_tests/test_ahi_hsd.py

```diff
@@ -139,7 +147,7 @@ def setUp(self, fromfile, np2str):
             with self.assertRaises(ValueError):
                 fh = AHIHSDFileHandler(None, {'segment_number': 8, 'total_segments': 10}, None, calib_mode='BAD_MODE')
 
-            fh = AHIHSDFileHandler(None, {'segment_number': 8, 'total_segments': 10}, None)
+            fh = AHIHSDFileHandler('test_file.bz2', {'segment_number': 8, 'total_segments': 10}, None)
```

Comment:
Could you add an assert that checks that `fh.filename` is being replaced ?

Reply:
Done!